src/Symfony/Component/HttpKernel/DataCollector/RequestDataCollector.php

```diff
@@ -263,11 +263,26 @@ public function getContent()
         return $this->data['content'];
     }
 
+    public function getPrettyJSONContent()
+    {
+        return json_encode(json_decode($this->getContent()), JSON_PRETTY_PRINT);
+    }
+
     public function getContentType()
     {
         return $this->data['content_type'];
     }
 
+    public function getRequestContentType()
```

Comment:
public method is unused, perhaps better to inline into `isRequestJSON` until proven needed

Reply:
I assume the fact I needed it would have proved it needed, but happy to inline if you think that's best.